src/Maestro/tests/Scenarios/azdoflow-batched.ps1

```diff
@@ -47,18 +45,18 @@ try {
     Write-Host "Running tests..."
     Write-Host
 
-    $source1RepoUri = Get-AzDO-RepoUri $source1RepoName
-    $source2RepoUri = Get-AzDO-RepoUri $source2RepoName
-    $targetRepoUri = Get-AzDO-RepoUri $targetRepoName
+    $source1RepoUri = Get-AzDO-RepoUri -repoName $source1RepoName
+    $source2RepoUri = Get-AzDO-RepoUri -repoName $source2RepoName
+    $targetRepoUri = Get-AzDO-RepoUri -repoName $targetRepoName
 
     Write-Host "Creating a test channel '$testChannelName'"
-    Darc-Add-Channel $testChannelName "test"
+    Darc-Add-Channel -channelName $testChannelName -classification "test"
 
     Write-Host "Adding a subscription from $source1RepoName to $targetRepoName"
-    $subscription1Id = Darc-Add-Subscription --channel $testChannelName --source-repo $source1RepoUri --target-repo $targetRepoUri --update-frequency none --target-branch $targetBranch --batchable
+    $subscription1Id = Darc-Add-Subscription -darcParams @( "--channel", "$testChannelName", "--source-repo", "$source1RepoUri", "--target-repo", "$targetRepoUri", "--update-frequency", "none", "--target-branch", "$targetBranch", "--batchable" )
```

Comment:
It seems a little weird to me for the caller of the function to have to create an array of [arg, value, arg, value ...] like the ones here, and it seems like a lot of work for what we want, (simply take the whole string after the command and run "darc" "command" "params")

Is there a way for the functions that are being called to do it without a complex regex, or putting the burden on the caller?

Reply:
It should be as simple as marking 'darcParams' with [Parameter(ValueFromRemainingArguments=$true) to capture all the args that are unbound. https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-6#valuefromremainingarguments-argument

Reply:
I'm trying switching to Alex's suggestion.

Reply:
This worked; tests are all still passing.